<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configuracion</class>
 <widget class="QWidget" name="configuracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuracion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #faf0d6;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QLabel" name="TipoDeVisualizacion">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #3C3C3C;
    font-size: 11px;
    font-weight: bold;
    padding: 6px;
    letter-spacing: 1px;
}</string>
     </property>
     <property name="text">
      <string>Tipo de visualización:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSlider" name="volumenSFXSlider">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* --- QSLIDER HORIZONTAL --- */
QSlider::groove:horizontal {
    border: 2px solid #C9A43A;              /* Contorno dorado */
    height: 10px;
    background: #F4E29B;                    /* Amarillo suave (igual que botones) */
    border-radius: 5px;
    margin: 0px 10px;
}

QSlider::handle:horizontal {
    background: qradialgradient(
        cx: 0.4, cy: 0.4,
        fx: 0.4, fy: 0.4,
        radius: 0.6,
        stop: 0 #FFE873,
        stop: 1 #C9A43A
    );
    border: 2px solid #5A4A00;
    width: 20px;
    height: 20px;
    border-radius: 10px;
    margin: -6px 0;
}

QSlider::handle:horizontal:hover {
    background: qradialgradient(
        cx: 0.4, cy: 0.4,
        fx: 0.4, fy: 0.4,
        radius: 0.6,
        stop: 0 #FFF3B0,
        stop: 1 #E5B937
    );
}

/* --- PARTE IZQUIERDA (LLENA) --- */
QSlider::sub-page:horizontal {
    background: #9BCB43;                    /* Verde suave tipo “Cargar datos” */
    border: 2px solid #5F8A2A;
    border-radius: 5px;
}

/* --- PARTE DERECHA (VACÍA) --- */
QSlider::add-page:horizontal {
    background: #EDE2AC;                    /* Beige claro */
    border: 2px solid #C9A43A;
    border-radius: 5px;
}

/* --- QSLIDER VERTICAL (por si lo usas) --- */
QSlider::groove:vertical {
    border: 2px solid #C9A43A;
    width: 10px;
    background: #F4E29B;
    border-radius: 5px;
    margin: 10px 0px;
}

QSlider::handle:vertical {
    background: qradialgradient(
        cx: 0.4, cy: 0.4,
        fx: 0.4, fy: 0.4,
        radius: 0.6,
        stop: 0 #FFE873,
        stop: 1 #C9A43A
    );
    border: 2px solid #5A4A00;
    height: 20px;
    width: 20px;
    border-radius: 10px;
    margin: 0 -6px;
}

QSlider::sub-page:vertical {
    background: #9BCB43;
    border: 2px solid #5F8A2A;
    border-radius: 5px;
}

QSlider::add-page:vertical {
    background: #EDE2AC;
    border: 2px solid #C9A43A;
    border-radius: 5px;
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="Resolucion">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #3C3C3C;
    font-size: 11px;
    font-weight: bold;
    padding: 6px;
    letter-spacing: 1px;
}</string>
     </property>
     <property name="text">
      <string>Resolución:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="guardar">
     <property name="styleSheet">
      <string notr="true">#guardar {
    background-color: #7DA23A;          /* Verde oliva oscuro */
    border: 3px solid #5C7A28;          /* Borde más oscuro para contraste */
    border-radius: 12px;
    color: #FDF5C9;                     /* Texto claro, cálido */
    font-size: 12px;
    font-weight: bold;
    padding: 8px 15px;
}

#guardar:hover {
    background-color: #8FBF42;          /* Un verde más vivo al pasar el ratón */
    border: 3px solid #6E9230;
}

#guardar:pressed {
    background-color: #6E8D2E;          /* Más oscuro al presionar */
    border: 3px solid #4F6B21;
}

#guardar:disabled {
    background-color: #A0A0A0;          /* Gris cuando está desactivado */
    color: #D8D8D8;
    border: 3px solid #808080;
}
</string>
     </property>
     <property name="text">
      <string>Guardar configuración</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="resolucion">
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* --- ComboBox principal --- */
QComboBox {
    font-family: &quot;Arial&quot;;
    font-size: 14px;
    font-weight: bold;
    padding: 6px 30px 6px 12px;
    min-height: 36px;
    border-radius: 10px;
    background: qlineargradient(
        x1: 0, y1: 0, x2: 0, y2: 1,
        stop: 0 rgba(246, 226, 143, 255),
        stop: 1 rgba(236, 198, 70, 255)
    );
    color: #2b2b2b;              /* texto oscuro, visible */
    border: 2px solid rgba(150, 110, 30, 0.6);
}

/* --- Flecha de desplegable --- */
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 36px;
    border: none;
    background: transparent;
}

QComboBox::down-arrow {
    width: 0;
    height: 0;
    margin-right: 10px;
    border-left: 6px solid transparent;
    border-right: 6px solid transparent;
    border-top: 8px solid rgba(70, 40, 10, 0.9); /* color de la flecha */
}

/* --- Cuando el combo está presionado --- */
QComboBox:on {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 0, y2: 1,
        stop: 0 rgba(240, 210, 110, 255),
        stop: 1 rgba(230, 180, 60, 255)
    );
}

/* --- Lista desplegable --- */
QComboBox QAbstractItemView {
    outline: 0;
    background: #fff9e6; /* tono beige claro */
    color: #2b2b2b;       /* texto oscuro */
    border: 1px solid rgba(150, 110, 30, 0.6);
    padding: 6px;
    border-radius: 6px;
    selection-background-color: rgba(245, 234, 180, 255);
    selection-color: black; /* texto visible al seleccionar */
}

/* --- Items de la lista --- */
QComboBox::item {
    color: #2b2b2b;
    background: transparent;
    padding: 4px 10px;
}

QComboBox::item:selected {
    background: rgba(245, 234, 180, 255);
    color: black;
}
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>640x480</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>960x540</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1280x720</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1920x1080</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="VolumenSFX">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #3C3C3C;
    font-size: 11px;
    font-weight: bold;
    padding: 6px;
    letter-spacing: 1px;
}</string>
     </property>
     <property name="text">
      <string>Volumen SFX:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSlider" name="volumenGeneralSlider">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* --- QSLIDER HORIZONTAL --- */
QSlider::groove:horizontal {
    border: 2px solid #C9A43A;              /* Contorno dorado */
    height: 10px;
    background: #F4E29B;                    /* Amarillo suave (igual que botones) */
    border-radius: 5px;
    margin: 0px 10px;
}

QSlider::handle:horizontal {
    background: qradialgradient(
        cx: 0.4, cy: 0.4,
        fx: 0.4, fy: 0.4,
        radius: 0.6,
        stop: 0 #FFE873,
        stop: 1 #C9A43A
    );
    border: 2px solid #5A4A00;
    width: 20px;
    height: 20px;
    border-radius: 10px;
    margin: -6px 0;
}

QSlider::handle:horizontal:hover {
    background: qradialgradient(
        cx: 0.4, cy: 0.4,
        fx: 0.4, fy: 0.4,
        radius: 0.6,
        stop: 0 #FFF3B0,
        stop: 1 #E5B937
    );
}

/* --- PARTE IZQUIERDA (LLENA) --- */
QSlider::sub-page:horizontal {
    background: #9BCB43;                    /* Verde suave tipo “Cargar datos” */
    border: 2px solid #5F8A2A;
    border-radius: 5px;
}

/* --- PARTE DERECHA (VACÍA) --- */
QSlider::add-page:horizontal {
    background: #EDE2AC;                    /* Beige claro */
    border: 2px solid #C9A43A;
    border-radius: 5px;
}

/* --- QSLIDER VERTICAL (por si lo usas) --- */
QSlider::groove:vertical {
    border: 2px solid #C9A43A;
    width: 10px;
    background: #F4E29B;
    border-radius: 5px;
    margin: 10px 0px;
}

QSlider::handle:vertical {
    background: qradialgradient(
        cx: 0.4, cy: 0.4,
        fx: 0.4, fy: 0.4,
        radius: 0.6,
        stop: 0 #FFE873,
        stop: 1 #C9A43A
    );
    border: 2px solid #5A4A00;
    height: 20px;
    width: 20px;
    border-radius: 10px;
    margin: 0 -6px;
}

QSlider::sub-page:vertical {
    background: #9BCB43;
    border: 2px solid #5F8A2A;
    border-radius: 5px;
}

QSlider::add-page:vertical {
    background: #EDE2AC;
    border: 2px solid #C9A43A;
    border-radius: 5px;
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="volumenSFX">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #3C3C3C;
    font-size: 11px;
    font-weight: bold;
    padding: 6px;
    letter-spacing: 1px;
}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="volumenGeneral">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #3C3C3C;
    font-size: 11px;
    font-weight: bold;
    padding: 6px;
    letter-spacing: 1px;
}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="VolumenGeneral">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #3C3C3C;
    font-size: 11px;
    font-weight: bold;
    padding: 6px;
    letter-spacing: 1px;
}</string>
     </property>
     <property name="text">
      <string>Volumen general:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QCheckBox" name="completa">
     <property name="styleSheet">
      <string notr="true">/* --- ESTILO GENERAL DE QCheckBox --- */
QCheckBox {
    spacing: 8px;
    font-size: 10px;
    color: #3B2E05;
    padding: 4px;
}

/* --- CHECKBOX SIN MARCAR --- */
QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 2px solid #C9A43A;        /* Borde dorado */
    border-radius: 4px;
    background-color: #F4E29B;        /* Amarillo suave */
}

/* --- CHECKBOX MARCADO --- */
QCheckBox::indicator:checked {
    image: none;
    background-color: #9BCB43;        /* Verde tipo “Cargar datos” */
    border: 2px solid #5F8A2A;
    border-radius: 4px;
}

/* --- CHECKBOX AL PASAR EL RATÓN --- */
QCheckBox::indicator:hover {
    background-color: #F9D95C;        /* Amarillo más brillante */
    border: 2px solid #E5B937;
}

/* --- CHECKBOX DESHABILITADO --- */
QCheckBox::indicator:disabled {
    background-color: #EDE2AC;
    border: 2px solid #B8A34F;
    opacity: 0.6;
}

QCheckBox:disabled {
    color: #8E8352;
}
</string>
     </property>
     <property name="text">
      <string>Completa</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="ventana">
     <property name="styleSheet">
      <string notr="true">/* --- ESTILO GENERAL DE QCheckBox --- */
QCheckBox {
    spacing: 8px;
    font-size: 10px;
    color: #3B2E05;
    padding: 4px;
}

/* --- CHECKBOX SIN MARCAR --- */
QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 2px solid #C9A43A;        /* Borde dorado */
    border-radius: 4px;
    background-color: #F4E29B;        /* Amarillo suave */
}

/* --- CHECKBOX MARCADO --- */
QCheckBox::indicator:checked {
    image: none;
    background-color: #9BCB43;        /* Verde tipo “Cargar datos” */
    border: 2px solid #5F8A2A;
    border-radius: 4px;
}

/* --- CHECKBOX AL PASAR EL RATÓN --- */
QCheckBox::indicator:hover {
    background-color: #F9D95C;        /* Amarillo más brillante */
    border: 2px solid #E5B937;
}

/* --- CHECKBOX DESHABILITADO --- */
QCheckBox::indicator:disabled {
    background-color: #EDE2AC;
    border: 2px solid #B8A34F;
    opacity: 0.6;
}

QCheckBox:disabled {
    color: #8E8352;
}
</string>
     </property>
     <property name="text">
      <string>Ventana </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
